config/runtime.exs

```diff
@@ -75,6 +75,21 @@ if config_env() in [:prod, :demo] do
       For example: http://localhost:8080
       """
 
+  amqp_url =
+    System.get_env("AMQP_URL") ||
+      raise """
+      environment variable AMQP_URL is missing.
+      For example: amqp://USER:PASSWORD@HOST
+      """
+
+  config :wanda, Wanda.Messaging.Adapters.AMQP,
```

Comment:
the first argument is the OTP app, here we are configuring `:trento` not `:wanda`

Reply:
damn copy/paste XD